test/src/xquery/maps/mapsLookup.xql

```diff
@@ -0,0 +1,23 @@
+xquery version "1.0";
```

Comment:
I've submitted a PR to fix that one. Isn't it odd that there's no error raised when using 3.x features when declaring the query as 1.0? 

```xquery
xquery version "1.0";

"a" || "b"
```

raises this error:

> Cannot compile xquery: exerr:EXXQDY0003 string concatenation operator is not available before XQuery 3.0 [at line 3, column 1, source: xquery version "1.0"; "a" || "b"]

but this doesn't:

```xquery
xquery version "1.0";

map { "a": "b" }
```